models/iaf_psc_alpha.cpp

```diff
@@ -382,15 +388,18 @@ iaf_psc_alpha::handle( SpikeEvent& e )
 {
   assert( e.get_delay_steps() > 0 );
 
+  const index input_buffer_slot = kernel().event_delivery_manager.get_modulo(
+    e.get_rel_delivery_steps( kernel().simulation_manager.get_slice_origin() ) );
```

Comment:
We ought to clean up this eyesore—you decide if you want to do it in this PR or create a new one. Can't we just have an `e.get_delivery_slot()` method and hide the remaining ugliness inside it?

Reply:
As stated above, I think the actual problem is that the `moduli_`-table machinery is not part of the ring-buffer class.